pkg/migrations/seq_num.go

```diff
@@ -1,30 +1,23 @@
 package migrations
 
 import (
-	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/migrations/internal"
-	"github.com/stackrox/rox/pkg/utils"
 )
 
 // CurrentDBVersionSeqNum is the current DB version number.
 // This must be incremented every time we write a migration.
 // It is a shared constant between central and the migrator binary.
 func CurrentDBVersionSeqNum() int {
-	// If PostgresDatastore is enabled, we consider all migrations to Postgres are following
-	// the current migrations.
-	// if features.PostgresDatastore.Enabled() {
-	//	return internal.CurrentDBVersionSeqNum + internal.PostgresDBVersionPlus
-	// }
+	if features.PostgresDatastore.Enabled() {
+		return internal.CurrentDBVersionSeqNum + internal.PostgresDBVersionPlus
+	}
 	return internal.CurrentDBVersionSeqNum
 }
 
 // CurrentDBVersionSeqNumWithoutPostgres is the current DB version number
```

Comment:
You are saying this should only be used in a testing environment, but you are using it in the migration.  So we probably need to clean up the comment.

Reply:
Will fix. Thanks!